docs/ECLLanguageReference/ECLR_mods/Basics-Constants.xml

```diff
@@ -20,6 +20,10 @@
     string, prepend a backslash (\). To include the backslash character (\) in
     a constant string, use two backslashes (\\) together.</para>
 
+    <para>The upper size limit for any string constant is 4GB (2^32), although
```

Comment:
I don't think the extra comment about running out of memory adds anything.  I'm not sure it is worth documenting the maximum length of a string constant at all, no one is going to have a string constant that long.